e2e/projects.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { Page, expect, test } from '@playwright/test'
+import { v4 as uuidv4 } from 'uuid'
+import { testAuth } from './fixtures/authenticatedUserPages'
+import { goToDashboard } from './helpers/testUser'
+
+const PROJECT_NAME = 'swc-e2e-project-' + uuidv4()
+
+async function createProject(page: Page, projectName: string) {
+  await page.getByLabel('Projects', { exact: true }).click()
+  await page.getByLabel('Create a New Project').click()
+
+  const projectNameInput = page.getByLabel('Project Name')
+  await projectNameInput.fill(projectName)
+  await expect(projectNameInput).toHaveValue(projectName)
+
+  await page.getByRole('button', { name: 'Save' }).click()
+}
+
+// Run multiple describes in parallel, but run tests inside each describe in order
+// ...tests within describe expect afterAll to be run with the same users, i.e. on the same worker
+// https://playwright.dev/docs/api/class-test#test-describe-configure
+test.describe.configure({ mode: 'serial' })
```

Comment:
Adding this block so that other test files can be [fully parallelized](https://playwright.dev/docs/test-parallel#parallelize-tests-in-a-single-file), i.e. tests within the same file can be split across workers, but this test describe will run serially on the same worker -- the `afterAll` should only run once, even if other tests are added to this describe. If this test describe was split across workers, then `afterAll` would be run for each test.

Reply:
Hmm...which do you think is more intuitive behavior, this, or serial-by-default and opt-in to parallel?

Reply:
Good question! I'm not sure -- is there a common default behavior in other test runners? I'll also pose the question to the team and see what everyone thinks!